update/ansible/playbook/tasks/roles/grafana/files/grafana.ini

```diff
@@ -24,7 +24,7 @@ format = console
 # enable gzip
 enable_gzip = true
 # The public facing domain name used to access grafana from a browser
-domain = 127.0.0.1
+domain = localhost
```

Comment:
as far as I see, we didn't have these lines at all

Reply:
Not really, they have always come from the config overrides, which has the same effect but is much les maintainable. Let me try the feature build - https://github.com/Percona-Lab/pmm-submodules/pull/3359

Reply:
no, in config overrides, we had that only if a public address is provided.

Reply:
yes, which is exactly when we need to override it, otherwise it should always be 127.0.0.1

Reply:
oh, wait I was wrong it was set only when PMM connected to the portal.

Reply:
yeah, and that's the only case when we need to change it